woocommerce/woocommerce-android
# Add Privacy Banner analytics events

Closes: #8970


### Description

This PR adds analytical events to the Privacy Banner.

### Testing instructions


Make sure Analytics is enabled.

1. Trigger the Privacy Banner (using instructions from previous PRs) -> check if `privacy_choices_banner_presented` is tracked
2. Tap on `Settings` -> check if `privacy_choices_banner_settings_button_tapped` is tracked
3. Tap on `Save` -> check if `privacy_choices_banner_save_button_tapped` is tracked
